main.go

```diff
@@ -92,6 +94,37 @@ func exportArtifacts(artifacts []gradle.Artifact, deployDir string) ([]string, e
 	return paths, nil
 }
 
+func filterUtilityVariants(variants []string) []string {
+	ignoredSuffixes := [...]string{"Classes", "Resources", "UnitTestClasses", "AndroidTestClasses", "AndroidTestResources"}
```

Comment:
What if somebody has a "classes" build type?

Reply:
The algorithm will remove it too. We might modify the condition to ignore the exact matches and filter only if it is a "real" suffix? Wdyt?

Reply:
I'm not sure we can come up with a non-exploitable solution parsing just the names. This was just a thought-provoking question. I see we a lot of times rely on name / string parsing in the CI which can go south easily. I think "classes" build-type is not likely so we should handle it if we get a bug report :)

Reply:
Sounds good to me, many thanks!